cli/ceph/ceph.py

```diff
@@ -108,3 +108,21 @@ def dashboard(self, **kw):
         if isinstance(out, tuple):
             return out[0].strip()
         return out
+
+    def logs(self, num, channel, level=None):
+        """
+        Run the ceph logs command to get the
+        cephadm logs
+        Args:
+            num(str): Number of logs required
+            level(str): Log level debug|info|sec|warn|error
+            channel(str): Get logs for channel cluster|audit|cephadm
+        """
+        cmd = f"{self.base_cmd} log last {num}"
+        if level:
+            cmd += f" {level} {channel}"
+        cmd += f" {channel}"
```

Comment:
This will add extra variable `channel` in case `level` is passed.

Reply:
The command doesn't works if don't pass channel and number of logs, so the only optional argument is level, but if the level need to be passed, it should be place before the channel and after number of logs, so I will make the changes accordingly. 

```
[ceph: root@ceph-vpap-2-xzdhew-node1-installer /]# ceph log last 5 cephadm
2024-06-11T07:37:39.217157+0000 mgr.ceph-vpap-2-xzdhew-node1-installer.gqfxeo (mgr.14217) 180 : cephadm [INF] Checking dashboard <-> RGW credentials
2024-06-11T07:39:09.694385+0000 mgr.ceph-vpap-2-xzdhew-node1-installer.gqfxeo (mgr.14217) 232 : cephadm [INF] Saving service mds.cephfs spec with placement count:2
2024-06-11T07:39:11.077168+0000 mgr.ceph-vpap-2-xzdhew-node1-installer.gqfxeo (mgr.14217) 235 : cephadm [INF] Saving service mds.cephfs spec with placement label:mds
2024-06-11T07:39:11.897472+0000 mgr.ceph-vpap-2-xzdhew-node1-installer.gqfxeo (mgr.14217) 237 : cephadm [INF] Deploying daemon mds.cephfs.ceph-vpap-2-xzdhew-node2.jghtie on ceph-vpap-2-xzdhew-node2
2024-06-11T07:39:13.121456+0000 mgr.ceph-vpap-2-xzdhew-node1-installer.gqfxeo (mgr.14217) 238 : cephadm [INF] Deploying daemon mds.cephfs.ceph-vpap-2-xzdhew-node3.qoymoo on ceph-vpap-2-xzdhew-node3

[ceph: root@ceph-vpap-2-xzdhew-node1-installer /]# ceph log last 5 cephadm info
Invalid command: unused arguments: ['info']
log last [<num:int>] [<level:debug|info|sec|warn|error>] [<channel:*|cluster|audit|cephadm>] :  print last few lines of the cluster log
Error EINVAL: invalid command

[ceph: root@ceph-vpap-2-xzdhew-node1-installer /]# ceph log last 5 info cephadm
2024-06-11T07:37:39.217157+0000 mgr.ceph-vpap-2-xzdhew-node1-installer.gqfxeo (mgr.14217) 180 : cephadm [INF] Checking dashboard <-> RGW credentials
2024-06-11T07:39:09.694385+0000 mgr.ceph-vpap-2-xzdhew-node1-installer.gqfxeo (mgr.14217) 232 : cephadm [INF] Saving service mds.cephfs spec with placement count:2
2024-06-11T07:39:11.077168+0000 mgr.ceph-vpap-2-xzdhew-node1-installer.gqfxeo (mgr.14217) 235 : cephadm [INF] Saving service mds.cephfs spec with placement label:mds
2024-06-11T07:39:11.897472+0000 mgr.ceph-vpap-2-xzdhew-node1-installer.gqfxeo (mgr.14217) 237 : cephadm [INF] Deploying daemon mds.cephfs.ceph-vpap-2-xzdhew-node2.jghtie on ceph-vpap-2-xzdhew-node2
2024-06-11T07:39:13.121456+0000 mgr.ceph-vpap-2-xzdhew-node1-installer.gqfxeo (mgr.14217) 238 : cephadm [INF] Deploying daemon mds.cephfs.ceph-vpap-2-xzdhew-node3.qoymoo on ceph-vpap-2-xzdhew-node3```